lib/index.js

```diff
@@ -771,6 +773,21 @@ class GalleryVideoArticle extends BaseAsset {
 
 exports.GalleryVideoArticle = GalleryVideoArticle;
 
+/** */
```

Comment:
Do we want to make it public API yet? If not, best to leave this out so it doesn't show up in the API reference. If yes, then document it :smile: (the `set_author()` method would need documentation as well)

If you document it, you'll also need to add a commit regenerating the documentation. Use `npm run docs:publish` before pushing your branch.

Reply:
Okay, will do

Reply:
Done